github-templates/CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing to (repo here)
+
+## Bugs, Feature Requests
+Submit an [issue](https://github.com/mobify/(repo here)/issues).
+
+## Pull Requests
+
+(Check the branching strategy docs for more info about Release vs Continuous Deployment: https://github.com/mobify/branching-strategy)
```

Comment:
I _think_ we could just have a small blurb here to follow our branching strategy and not duplicate the steps here. The steps below will just add maintenance burden as our branching strategy evolves.